cmd/server/command/start.go

```diff
@@ -169,7 +173,18 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 
 			var s3storageSvc *s3storage.Service
 			if startOptions.s3storage.S3BucketName != "" {
-				s3storageSvc, err = s3storage.New(startOptions.s3storage.S3BucketName, tracer)
+				region := "eu-west-1"
```

Comment:
I think we should make this one configurable?

Reply:
I didn't want to change anything in this behaviour, but I agree. Will follow up with some additional fixes I think.

Another things is to replace the existing integration tests with the new fake S3 client to make them runable in CI.

Reply:
I prepared the removal of the live S3 fixtures and it turns out it was super simple so I have added that to this branch.